jaxrs/src/main/java/io/airlift/jaxrs/JaxrsBinder.java

```diff
@@ -43,6 +43,12 @@ public void bind(Key<?> targetKey)
         resourceBinder.addBinding().to(targetKey).in(SINGLETON);
     }
 
+    public JaxrsBinder withStandardExceptionMapper()
+    {
+        bind(JsonParsingExceptionMapper.class);
```

Comment:
I don't follow. I thought this was going to be a default behavior but this is an opt-in, not opt-out feature

Reply:
Oh! Opt out is better. I wasn’t sure. I’ll change it.